sim/src/test/scala/midasexamples/TutorialSuite.scala

```diff
@@ -112,5 +112,7 @@ class NarrowPrintfModuleF1Test extends TutorialSuite("NarrowPrintfModule",
   simulationArgs = Seq("+print-no-cycle-prefix", "+print-file=synthprinttest.out")) {
   diffSynthesizedPrints("synthprinttest.out")
 }
+
 // MIDAS 2.0 compiler tests
 class WireInterconnectF1Test extends TutorialSuite("WireInterconnect")
+class VerilogAccumulatorF1Test extends TutorialSuite("VerilogAccumulator")
```

Comment:
Did you intentionally neglect the chisel accumulator?

Reply:
Yes but I can add it if that makes sense.

Reply:
You should either add it or remove it all together. I'm always down for less code.

Reply:
Removed